src/style.css

```
@@ -0,0 +1,119 @@
+* {
+  box-sizing: border-box;
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
+.Html-Body {
+  margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.Card {
```

```suggestion
.card {
```
Назви класів починаються з маленької літери

---

а як же стиль Реакт?
https://ru.bem.info/methodology/naming-convention/#%D1%81%D1%82%D0%B8%D0%BB%D1%8C-react

---

Стиль Реакт, как правило, применяется в React, а не в обычном html/css